<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditInvoiceForm</class>
 <widget class="QDialog" name="EditInvoiceForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>486</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>425</width>
    <height>486</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>425</width>
    <height>486</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit invoice</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #F5F8FA;
color: #14171A;
font-family:&quot;Segoe UI&quot;,Roboto,&quot;Helvetica Neue&quot;,Arial,&quot;Noto Sans&quot;,sans-serif,&quot;Apple Color Emoji&quot;,&quot;Segoe UI Emoji&quot;,&quot;Segoe UI Symbol&quot;,&quot;Noto Color Emoji&quot;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QLabel" name="headerTitle">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 18px;
font-weight:500;
padding-left:4px;
</string>
     </property>
     <property name="text">
      <string>Edit the payment</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="paymentBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Payment details</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QWidget" name="studentWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="studentLabel">
           <property name="styleSheet">
            <string notr="true">padding-bottom: 6px;
font-size:14px;</string>
           </property>
           <property name="text">
            <string>From student:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="studentCombo">
           <property name="styleSheet">
            <string notr="true">QComboBox{
	padding: 6px 4px;
	border: 1px solid #aab8c2;
	border-radius: 2px;
}
QComboBox:hover, QComboBox:focus{
	border: 1px solid #14171a;
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="descWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="descLabel">
           <property name="styleSheet">
            <string notr="true">padding-bottom: 6px;
font-size:14px;</string>
           </property>
           <property name="text">
            <string>Description:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="descText">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QTextEdit{
	padding: 6px 4px;
	border: 1px solid #aab8c2;
	border-radius: 2px;
}
QTextEdit:hover, QTextEdit:focus{
	border: 1px solid #14171a;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="acceptRichText">
            <bool>false</bool>
           </property>
           <property name="placeholderText">
            <string>Optional</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="amountWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="amountLabel">
           <property name="styleSheet">
            <string notr="true">padding-bottom: 6px;
font-size:14px;</string>
           </property>
           <property name="text">
            <string>Amount of payment:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="amountSpin">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="suffix">
            <string> PLN</string>
           </property>
           <property name="minimum">
            <double>0.010000000000000</double>
           </property>
           <property name="maximum">
            <double>99999.990000000005239</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="untilWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="untilLabel">
           <property name="styleSheet">
            <string notr="true">padding-bottom: 6px;
font-size:14px;</string>
           </property>
           <property name="text">
            <string>Paid to:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="untilDateInput">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="specialValueText">
            <string/>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="styleSheet">
      <string notr="true">QPushButton {
background: #616161;
border-radius: 2px;
color: #fff;
padding: 8px 16px;
}</string>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Discard|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
